src/olympia/amo/admin.py

```diff
@@ -41,6 +45,50 @@ class AMOModelAdminChangeList(ChangeList):
 
     search_form_class = AMOModelAdminChangeListSearchForm
 
+    def __init__(self, request, *args, **kwargs):
+        super().__init__(request, *args, **kwargs)
+        # django's ChangeList does:
+        # self.params = dict(request.GET.items())
+        # But we want to keep a QueryDict to not lose parameters present
+        # multiple times.
+        self.params = request.GET.copy()
```

Comment:
because `QueryDict` can still be manipulated as a `dict`, it's fine. Note that we were already using this code in `ScannerResultAdmin`'s `ChangeList`, I just moved it to make it available to more admin tools that have filters that can be present multiple times.